src/test/libcephfs/snapdiff.cc

```diff
@@ -56,7 +59,58 @@ class TestMount {
     return ceph_conf_get(cmount, option, buf, len);
   }
 
-  string make_file_path(const char* relpath) {
+  int rank0_config_set(const char * var, const char * val) {
+    char *outb,*outs;
+    size_t outb_len, outs_len;
+
+    char cmd[1024];
+    snprintf(cmd, sizeof(cmd) / sizeof(char), R"({"prefix":"config set","var":"%s","val":["%s"]})", var, val);
+    const char* cmdv[] = {cmd};
+    int status = ceph_mds_command(cmount, "0", cmdv, 1, nullptr, 0, &outb, &outb_len, &outs, &outs_len);
+    if (status < 0) {
+      std::cout << "couldn't set config option '" << var << "': " << strerror(-status) << " outs: " << outs << std::endl;
+    }
+    return status;
+  }
+
+  int rank0_config_unset(const char *var){
```

Comment:
Probably overload `rank0_config_set` to handle unset cases when `val == nullptr`?